Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM continuumio/miniconda
 
-RUN pip install numpy pandas flask pygal smalluuid zipstream python-dateutil gitpython scikit-learn
-
 WORKDIR /app
 
 ADD . /app
+
+RUN pip install -r dev-requirements.txt && pip install -r tox-requirements.txt && pip install -e .
```

Comment:
Can we actually collapse all of these RUN's to a single RUN?

Also a nit on the `&&`, can we have each `&&` on the new line?